README.md

```diff
@@ -33,6 +33,12 @@ Generate OWASP ASVS Sarif file
 generate-owasp --source-file-path "OWASP Application Security Verification Standard 4.0.2-en.csv" --target-file-path "OWASP_ASVS_v4.0.2.sarif" --version "4.0.2"
```

Comment:
>generate-owasp [](http://example.com/codeflow?start=0&length=14)

also missing type argument

Reply:
yes, the previous commit was just the merge from master, just fixed